types/backend/index.d.ts

```diff
@@ -1 +1,4 @@
-/// <reference path="../common/declaration.d.ts" />
\ No newline at end of file
+/// <reference path="../common/declaration.d.ts" />
```

Comment:
can you move the changes regarding $w compilation errors to a different commit ?